nncf/openvino/graph/model_transformer.py

```diff
@@ -431,3 +433,34 @@ def _insert_inplace_operation(
             new_node = transformation.inplace_op_fn(output.get_node(), output.get_index())
             return (new_node.output(fn_output_port_id), fn_output_port_id)
         raise RuntimeError(f"Transform type {transform_type} is not supported")
+
+    @staticmethod
+    def _apply_bias_insertion_transformations(
+        model: ov.Model, transformations: List[OVBiasInsertionCommand]
+    ) -> ov.Model:
+        """
+        Inserts bias operation after corresponding layer.
+
+        :param transformations: List of the bias insertion transformations.
+        :returns: Transformed model with null biases.
+        """
+        name_to_node_mapping = OVModelTransformer._get_name_to_node_mapping(model)
+        for transformation in transformations:
+            node_name = transformation.target_point.target_node_name
+            node = name_to_node_mapping[node_name]
+            node_shape = node.output(0).partial_shape.get_max_shape()
+            node_output_port = node.output(transformation.target_point.port_id)
+            node_output_source_ports = node_output_port.get_target_inputs()
+
+            bias_shape = [1] * len(node_shape)
+            bias_shape[1] = node_shape[1]
+            const_value = np.zeros(bias_shape, dtype=node.get_element_type().to_dtype())
+            bias_const_node = opset.constant(const_value, dtype=node.get_element_type())
+            bias_const_output_port = bias_const_node.output(0)
+
+            add_node = opset.add(node_output_port, bias_const_output_port, name=f"{node_name}/add_")
```

Comment:
Can we specify in node name that this add is dummy and adds 0?

Reply:
We can, but why do we need it?

Reply:
Just for the sake of explicitness and to prevent naming conflicts

Reply:
Done.